secure/key-manager/src/tests.rs

```diff
@@ -163,34 +162,38 @@ impl LibraInterface for TestLibraInterface {
     }
 }
 
-fn execute_and_commit(node: &Node, mut block: Vec<Transaction>) {
+fn execute_and_commit(node: &mut Node, mut block: Vec<Transaction>) {
     let block_id = HashValue::zero();
     let startup_info = node.storage.get_startup_info().unwrap().unwrap();
     let clock = startup_info.committed_tree_state.version + 1;
-    let committed_trees = ExecutedTrees::from(startup_info.committed_tree_state);
 
     // This will set the current clock to be equal to the current version + 1, this guarantees that
     // the clock is always refreshed for this set of transactions.
     let block_metadata = BlockMetadata::new(block_id, 0, clock, vec![], node.account);
     let prologue = Transaction::BlockMetadata(block_metadata);
     block.insert(0, prologue);
 
-    let output = node
+    let result = node
         .executor
-        .execute_block(block_id, block.clone(), &committed_trees, &committed_trees)
+        .execute_block((block_id, block), node.executor.committed_block_id())
```

Comment:
could we not have made this into another PR?

Reply:
how...
I think if we made this into another PR, this PR will not pass CI. This test will fail since it calls executor's api.